common/src/main/java/com/wynntils/wc/utils/scoreboard/objectives/ObjectiveManager.java

```diff
@@ -83,7 +83,9 @@ public static List<WynnObjective> getObjectives() {
 
     @Override
     public void onSegmentChange(Segment newValue, ScoreboardManager.SegmentType segmentType) {
-        List<WynnObjective> objectives = reparseObjectives(newValue);
+        List<WynnObjective> objectives = reparseObjectives(newValue).stream()
+                .filter(wynnObjective -> wynnObjective.getScore() != wynnObjective.getMaxScore())
```

Comment:
```suggestion
                .filter(wynnObjective -> wynnObjective.getScore() < wynnObjective.getMaxScore())
```
would be clearer I think